Tropos.xcodeproj/project.pbxproj

```diff
@@ -998,7 +1005,6 @@
 				COPY_PHASE_STRIP = YES;
 				DEFINES_MODULE = YES;
 				DSTROOT = /tmp/xcodeproj.dst;
-				EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
```

Comment:
Because Cocoapods complained about it. I deleted the build setting to set it's back to it's default. Cocoapods will set it to the value it needs using it's xcconfig.